### PR TITLE
Fix app update electron

### PR DESCRIPTION
**Pull request by afaur**

### CODE REVIEW

**Comment by afaur** [comment]
There is one issue here.. basically to run the tests successfully `clipboard.js` needs to have `electron` in the package.json deps, however, adding `electron` into the deps messes up the `npm run start` because `electron-prebuilt` provides `electron` to the bootstrapped app.  

As it currently is the test are passing with the code committed but the `npm run start` command will fail if you pull this and run it since `electron` is included in the package.json.

File: lib/app.js [hunk 1]
```diff
@@ -1,12 +1,26 @@
 'use strict';
 
+// Require base electron
```
Comment (blainesch) [inline lib/app.js]: Do we need all these comments?

File: lib/app.js [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
 const electron = require('electron');
+
+// Module to control application life.
 const app = electron.app;  // Module to control application life.
+
+// Ref to chromium browser
+const BrowserWindow = electron.BrowserWindow;  // Module to create native browser window.
```
Comment (blainesch) [inline lib/app.js]: Double comment lol

File: lib/plugin/output/copyToClipboard.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const cuid = require('cuid');
+const electron = require('electron');
```
Comment (blainesch) [inline lib/plugin/output/copyToClipboard.js]: I think there's a way to overwrite electron when it's required, so we can overwrite the "clipboard" for testing.. make it a spy.

File: package.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/tinytacoteam/zazu",
   "devDependencies": {
-    "electron-prebuilt": "^0.35.0",
+    "electron-prebuilt": "^0.37.2",
```
Comment (blainesch) [inline package.json]: If this requires electron, do we still need it as a dependency?

**Pull request closed without merging**